User-Help/Voting.md

```diff
@@ -0,0 +1,40 @@
+| URL slug | Category |
+|:--------:|:--------:|
+| voting | Guidance |
+
+# Why should I vote?
+
+Voting is the primary indicator of valuable content.  Vote up posts that add to the community, and vote down ones that detract from it.  (Many posts do neither; you don't have to vote on everything.)
+
+Voting influences a post's score.  Post score contributes to earning abilities.  When you vote, you are, in a sense, saying "I would like the community to get more (or less) of this kind of post".
```

Comment:
When you vote, you are in a sense saying, "I would like the community..." 

More natural comma placement?